src/v4/options.rs

```diff
@@ -1199,7 +1217,8 @@ impl Encodable for DhcpOption {
             | RouterSolicitationAddr(addr)
             | RequestedIpAddress(addr)
             | ServerIdentifier(addr)
-            | SubnetSelection(addr) => {
+            | SubnetSelection(addr)
+            | TFTPServerAdress(addr) => {
```

Comment:
That
```text
-         | SubnetSelection(addr) => {
+         | SubnetSelection(addr)
+         | TFTPServerAdress(addr) => {
```
how that curly bracket open, `{`, did **not** stay with the `SubnetSelection()` feels strange.

Reply:
As long as it's gone through rustfmt!